oxd-common/src/main/java/org/gluu/oxd/common/CoreUtils.java

```diff
@@ -260,4 +283,8 @@ public static String cleanUpLog(String log) {
         }
         return log;
     }
+
+    private static boolean isSafePort(Integer input) {
```

Comment:
Lets now put validation logic here. During startup validate parameters and if it's not valid throw error.

Reply:
Added validation logic for proxy `host`. `port` and `protocol` are not mandatory.